### PR TITLE
Add ShiftedArrays

### PR DESCRIPTION
**Pull request by nalimilan**
README.md assumes ShiftedArrays is going to move to JuliaStats.

### CODE REVIEW

**Comment by piever** [comment]
OK, happy to move the repository as soon as the decision on where it goes (JuliaArrays or JuliaStats) is finalized.I may be mildly in favor of moving it to JuliaArrays, of which I'm already a member, but am fine with either option.

**Comment by nalimilan** [comment]
Ah, right, I also think JuliaArrays is more appropriate. I've asked on Slack.

**Comment by piever** [comment]
UPDATE: I've just transferred the repository at https://github.com/JuliaArrays/ShiftedArrays.jl

**Comment by nalimilan** [comment]
Cool. Let's merge? CI failures are timeouts (there are too many tests...).

File: src/StatsKit.jl [hunk 1]
```diff
@@ -12,6 +12,7 @@ module StatsKit
     @reexport using KernelDensity
     @reexport using Loess
     @reexport using MultivariateStats
+    @reexport using ShiftedArrays
```
Comment (piever) [inline src/StatsKit.jl]: Shouldn't it be added to REQUIRE / Project.toml as well?

Comment (nalimilan) [reply src/StatsKit.jl]: Woops, of course. Let's first move to Project.toml: https://github.com/JuliaStats/StatsKit.jl/pull/23

Comment (nalimilan) [reply src/StatsKit.jl]: Done.

**Comment by nalimilan** [comment]
OK to merge?

**Comment by nalimilan** [comment]
Actually I'm afraid I'm going to have to revert this, as TimeSeries already exports `lag`, lead` and `diff`: https://juliastats.org/TimeSeries.jl/stable/apply/#lag-1

It only provides methods for `TimeArray` though (and it's a not an `AbstractArray`), so maybe we can define these functions e.g. in DataAPI so that both packages can extend them.

Cc: @iblis17

**Comment by iblis17** [comment]
> It only provides methods for TimeArray though (and it's a not an AbstractArray), so maybe we can define these functions e.g. in DataAPI so that both packages can extend them.

I'm okay if we define them in DataAPI.

**Pull request merged**